front/pages/w/[wId]/ds/[name]/settings.tsx

```diff
@@ -20,9 +20,9 @@ import { UserType, WorkspaceType } from "@app/types/user";
 
 const {
   GA_TRACKING_ID = "",
-  NANGO_SLACK_CONNECTOR_ID,
-  NANGO_NOTION_CONNECTOR_ID,
-  NANGO_PUBLIC_KEY,
+  NANGO_SLACK_CONNECTOR_ID = "",
```

Comment:
I think we agreed that we don't want to crash the app at import time just because we don't have the nango params defined.